usi/src/main/proto/usi.proto

```diff
@@ -36,6 +36,7 @@ enum SwitchModel {
   ALLIED_TELESIS_X230 = 0;
   CISCO_9300 = 1;
   OVS_SWITCH = 2;
+  FAUX_SWITCH = 3;
```

Comment:
what's this for?

Reply:
There's a setup where there is no switch that can be controlled (it's OVS but not reachable)... and that's what this field indicated.  but, this is not the right place to handle that, so I put a check in at a different layer to detect that connection and not allow the connect RPC call at all.